### PR TITLE
V01 master kube proto v2

### PR DESCRIPTION
**Pull request by diclophis**
```
$ kubectl get pods
NAME                          READY     STATUS        RESTARTS   AGE
redis-854b8b7b78-7kp8g        1/1       Running       0          6s
spawn-world-and-coins-nvxxf   1/1       Terminating   0          4m
spawn-world-and-coins-qpmd2   1/1       Running       0          5s
zerozero-855b9496b8-tq985     1/1       Running       0          6s
$ kubectl logs -f spawn-world-and-coins-qpmd2
+ zerozero-new-world
newWorld.go: resetting world
newWorld.go: creating new world
+ zerozero-spawn-coins
^C
$ kubectl logs -f zerozero-855b9496b8-tq985
factory.go: No TypeFactory Instance found. Creating.
factory.go: Finished registering DboFactoring.
```

### CODE REVIEW

**Comment by diclophis** [comment]
screen shot 2018-09-16 at 10 51 57 pm

Why does the server use so much CPU?

**Comment by bennapp** [comment]
> Why does the server use so much CPU?

I have no idea. TODO: figure that out.

**Comment by bennapp** [comment]
I am getting this running the `front-end`.

```
index.js:87 WebSocket connection to 'ws://localhost:8081/ws' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED
```

**Comment by diclophis** [comment]
the CPU burn was some bad redis config causing i think a retry loop

**Comment by diclophis** [comment]
are you trying to run `kube.yml` and getting the err conn refused error? or normally?

**Comment by bennapp** [comment]
running normally, w/ `yarn run assets`.

**Comment by diclophis** [comment]
screen shot 2018-09-18 at 9 09 55 pm

**Comment by bennapp** [comment]
Those are some hot rocks and coins.

**Comment by diclophis** [comment]
screen shot 2018-09-18 at 11 56 23 pm
screen shot 2018-09-18 at 11 56 07 pm

**Comment by diclophis** [comment]
should be good to go on the phaser.js client now

File: front-end/src/client/index.js [hunk 1]
```diff
@@ -90,9 +90,13 @@ function create() {
     conn.onclose = function (event) {
       console.log("Connection closed.");
     };
+
     conn.onmessage = function (event) {
-      console.log(msgpack.decode(event.data));
-      self.gameStateUpdate(event.data);
+      //need to cast the raw buffer to a sequence of typed elements
+      var typedData = new Uint8Array(event.data)
```
Comment (bennapp) [inline front-end/src/client/index.js]: Nice, ok we needed this bit too.

**Pull request merged**